config/locales/views/saving_goals/ca.yml

```
@@ -0,0 +1,93 @@
+      target: "Objectiu: %{amount}"
+      deadline: "Data límit: %{date}"
+      on_track: "A temps"
+      behind: "Amb retard"
+      contributions: "Contribucions"
+      actions: "Accions"
+      edit: "Editar objectiu"
+      delete: "Eliminar objectiu"
+      target_date: "Data objectiu: %{date}"
+      current_balance: "Balanç actual"
+      of_target: "de l'objectiu de %{amount}"
+      on_track: "A temps"
```

YAMLlint (1.38.0)

[error] duplication of key "on_track" in mapping

(key-duplicates)